android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java

```diff
@@ -69,6 +78,14 @@ public static void launch(Context context, HashMap<String, Object> extraData) {
 
         ComponentName componentName;
 
+        if (ContextCompat.checkSelfPermission(context, POST_NOTIFICATIONS) != PERMISSION_GRANTED) {
+            ActivityCompat.requestPermissions(
```

Comment:
Have you tested this doesn't interfere with the permissions listener in the delegate?